src/app/modules/sgid/sgid.service.ts

```diff
@@ -129,6 +158,11 @@ export class SgidService {
     )
   }
 
+  /**
+   * Create a JWT based on the userinfo from sgID
+   * @param {{ 'myinfo.nric_number': string }} data - the userinfo as obtained from sgID
+   * @param {Boolean} rememberMe - determines how long the JWT is valid for
+   */
   createJWT(
```

Comment:
nit: change name to `createJwt` instead to follow the current (unspoken and loosely enforced 😢 ) naming conventions in this repo?